CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# Contributing to Animate.css
+Thanks for your interest in contributing to Animate.css! Before contributing, please make sure you understand the guidelines provided here. Animate.css is widely used, so it’s important to maintain a high level of quality and to contribute with the interests of the community in mind.
+
+## Design Guidelines
+Animations, like many facets of visual and interaction design, can be highly subjective. Maintaining a consistent library of animations in an active community can be difficult; these design guidelines are designed to help encourage thoughtful criticism of new animations that are proposed for Animate.css.
+
+The animations in Animate.css should follow a few key principles:
+
+- **Animations should be subtle.** Avoid creating animations that involve large translations, or span a natural duration of longer than 1 second.
+- **Animations should be tolerable.** Related to subtlety, animations should be tolerable—seeing them repeatedly should not become too annoying or overbearing.
+- **Animations should not interfere with document flow or control/input availability.** In other words, the absence of an animation should never reduce usability of a product: they should be non-critical and seen as “progressive enhancements”. Avoid animations that change properties such as `position` or `display`.
```

Comment:
I think we can change "such as `position` or `display`"  for: "other than `transform` and `opacity`.